tracer/test/test-applications/security/Samples.Security.AspNetCore5/Controllers/IastController.cs

```diff
@@ -443,6 +443,34 @@ public ActionResult Ssrf(string url, string host)
             return Content(result, "text/html");
         }
 
+        [HttpGet("TBV")]
+        [Route("TBV")]
+        public ActionResult Tbv(string name, string value)
+        {
+            string result = string.Empty;
+            try
+            {
+                if (HttpContext.Session == null)
+                {
+                    result = "No session";
+                }
+                else
+                {
+                    HttpContext.Session.SetString(name, value);
+                    HttpContext.Session.SetInt32(name + "-" + value, 42);
```

Comment:
Ah ok, I guess this answers my previous comment - we use both values twice 🙂

Reply:
The PR is [here](https://github.com/DataDog/dd-go/pull/110822)
So this is not needed anymore then?